test/extended/builds/gitauth.go

```diff
@@ -1,6 +1,6 @@
 package builds
 
-// these tests are diabled because the xip.io dns hook was proving way too unreliable;
+// these tests are diabled because the nip.io dns hook was proving way too unreliable;
```

Comment:
@csrwng How about enabling it in this PR and running tests a few times to see if it solves the issue? That would give more confidence that this change actually switches to a more stable wildcard DNS service?